crates/holochain_types/src/dna/dna_bundle.rs

```diff
@@ -357,4 +390,102 @@ mod tests {
             SerializedBytes::try_from(properties).unwrap()
         );
     }
+
```

Comment:
Unit tests that inline zomes cannot be bundled into a dna bundle and that a dna manifest cannot be constructed from a `DnaDef` containing inline zomes.

Reply:
Neat tests, writing tests for panics is a useful thing to do. Just because we don't want them to happen doesn't mean we shouldn't check when they are needed :D